src/other/index.jl

```diff
@@ -112,7 +112,10 @@ Base.getindex(x::Index, idx::Symbol) = x.lookup[idx]
 Base.getindex(x::AbstractIndex, idx::Real) = Int(idx)
 Base.getindex(x::AbstractIndex, idx::AbstractVector{Union{Bool, Missing}}) =
     getindex(x, collect(Missings.replace(idx, false)))
```

Comment:
This constructor needs the same check, right? Though I guess we can leave it that way if you remove it soon.

Reply:
this one will call the one I changed so it should be safe (and as you say I plan to remove it anyway).
Does removing of handling of `Missing` require deprecation period?

Reply:
Ah, indeed. Dropping support for missing values should indeed go through a deprecation, but that's a different issue, right?

Reply:
yes